src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -38,11 +39,14 @@ internal class StdIOTransport : IConnectionTransport, IDisposable
         private readonly List<byte> _data = new();
         private int? _currentMessageSize;
 
-        internal StdIOTransport(Process process)
+        internal StdIOTransport()
         {
-            _process = process;
-            process.ErrorDataReceived += (_, e) => LogReceived?.Invoke(this, new(e.Data));
-            process.BeginErrorReadLine();
+            _process = GetProcess();
+            _process.StartInfo.Arguments = "run-driver";
+            _process.Start();
+            _process.Exited += (_, _) => Close("Process exited");
```

Comment:
Hm, you have this, but then you also have the same in `[TearDown]`. That might also be the reason for failures, i.e., if that process dies (legitimately), this gets triggered, and then the tear down happens & re-triggers it. 

You can also just have a `Close` method on the `BrowserServer` and it checks it hasn't been called yet, if you wanted to.

Reply:
I don't this will affect since we are spawning two different servers one is for connect tests & which is separate from the process object I don't think this can be an issue. Since the above close will close the server for execution & the test will close the process of the test playwright server.

Reply:
Have updated tests it was due to the playwright process not getting started. Have fixed tests